lib/stripe.ts

```diff
@@ -0,0 +1,40 @@
+import Stripe from 'stripe';
+import env from '@/lib/env';
+import { updateTeam } from 'models/team';
+
+export const stripe = new Stripe(env.stripe.secretKey ?? '', {
+  // https://github.com/stripe/stripe-node#configuration
+  apiVersion: '2022-11-15',
+  // Register this as an official Stripe plugin.
+  // https://stripe.com/docs/building-plugins#setappinfo
+  appInfo: {
+    name: 'saas-starter-kit',
+    version: '0.1.0',
+  },
```

Comment:
is this really needed? This option is deprecated https://stripe.com/docs/building-plugins#setappinfo

Reply:
fixed